src/posts/abstracting-the-frontend/index.md

```diff
@@ -0,0 +1,124 @@
+---
+path: '/blog/abstracting-the-front-end'
+title: 'Abstracting The Front End'
+date: '2020-03-15'
+featuredImage: './cover.jpg'
+---
+
+We so rarely talk about the front end in abstract terms. It's all _"how to x in React"_ or _"Scaling apps in my favourite framework"._
+
+Let's take a moment and talk about how we could split up our front-end concerns on a higher level,
+agnostic of any framework or technology, so that we can avoid distractions when dealing with the aforementioned challenges.
+
+### The Front End
+
+What is the front end?
+
+It is my opinion that today's front-end developer should harness the skill to provide a user with a visual interface
```

Comment:
Vad innebär det att "provide a user with a visual interface
to interact with the product one is working on." ? Ta resonemanget det en nivå tydligare. Jag brukar säga att User interfacet är integrationenslagret mellan en mänsklig hjärna och en CPU.

Reply:
Jag håller med att meningen är inte tillräckligt enkel. Kikar på det.